### ISSUE TITLE
Add icon on title bar

### ISSUE THREAD

**Issue by RaphaelSilv** [josephhutch/aether#53]
Hi folks, I've been struggling with this simple task.
I've followed everything on _aether_ [README](https://github.com/josephhutch/aether) in the **Favicon** section and it still not working.

I realized my `layout` folder was empty, so I copied everything from `themes/aether/layouts.`
So far, so good.

 From `layouts/partials` my `head.md` is like so:
```html
<head>
  <meta charset="utf-8" />
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <title>{{ if not .IsHome }}{{with .Title }}{{ . }} &middot; {{ end }}{{ end }}{{ .Site.Title }}</title>
  <meta name="description" content="{{if .IsHome}}{{ $.Site.Params.description }}{{else}}{{.Description}}{{end}}" />
  {{ if (fileExists "static/favicon.ico") -}}
    <link rel="shortcut icon" href="/favicon.ico">
  {{ end -}}
  {{ if (fileExists "static/favicon-32x32.png") -}}
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
  {{ end -}}
  {{ if (fileExists "static/favicon-16x16.png") -}}
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
  {{ end -}}
  {{ if (fileExists "static/site.webmanifest") -}}
  <link rel="manifest" href="/site.webmanifest">
  {{ end -}}
  {{ if (fileExists "static/safari-pinned-tab.svg") -}}
    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
  {{ end -}}
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css" integrity="sha384-9eLZqc9ds8eNjO3TmqPeYcDj8n+Qfa4nuSiGYa6DjLNcv9BtN69ZIulL9+8CqC9Y" crossorigin="anonymous">
  {{ $mainStyle := resources.Get "css/style.css" }}
  {{ $xcodeStyle := resources.Get "css/xcode.css" }}
  {{ $fontStyle := resources.Get "css/latolatinfonts.css" }}
  {{ $overrideStyle := resources.Get "css/override.css" }}
  {{ $style := slice $xcodeStyle $fontStyle $mainStyle $overrideStyle | resources.Concat "css/concated.css" | minify }}
  <link href="{{ $style.Permalink }}" rel="stylesheet">
  {{ range .AlternativeOutputFormats -}}
    {{ printf `<link rel="%s" type="%s" href="%s" title="%s" />` .Rel .MediaType.Type .Permalink $.Site.Title | safeHTML }}
  {{ end -}}
  {{ template "_internal/google_analytics.html" . }}
  {{ if isset $.Site.Params "bgimg" -}}
  <style>
    body {
      background: #ecedef url("{{ $.Site.Params.bgimg | absURL | safeCSS }}") repeat;
    }
  </style>
  {{- end }}
</head>
---------
```

Inside `static` I've placed all the images extracted from the **favicon generator** downloaded folder.
Like so:
```sh
android-chrome-192x192.png  apple-touch-icon.png  favicon-16x16.png  favicon.ico  mstile-150x150.png
android-chrome-512x512.png  browserconfig.xml     favicon-32x32.png  img          site.webmanifest
```
Also, my config.toml:

```cfg
baseURL = "https://raphaelsilv.github.io/raphael/"
languageCode = "The language code for the language the website is written in"
title = "Raphael Ramos | Code Insights"
theme = "aether"
googleAnalytics = "Your google analytics tracking ID - optional"
disqusShortname = "basquiatraphaeu"
tags = ["Blog", "Personal", "Raphael", "Shell", "Simple", "Code", "Programmer", "Technical", "Github", "UFSC", "C++", "Shell"]
features = ["Responsive", "Highlight.js", "KaTeX", "Google analytics", "Disqus"]
min_version = "0.01"
publishDir = "docs"

[params]
brand = "By: Raphael Ramos"
description = "The website's description"
homeimg = "img/fancy-pants.jpg"
bgimg = "img/ignasi_pattern_s.png"

[author]
name = "Raphael Ramos"
homepage = "https://www.linkedin.com/in/raphaelramosdasilva/"

[markup.goldmark.renderer]
unsafe= true---
```

What am I missing, guys?

**Comment by josephhutch** [comment]
I found the issue.  I'll issue a fix shortly.

**Comment by RaphaelSilv** [comment]
Looking forward, @josephhutch. Great job written `aether` btw.

**Comment by josephhutch** [comment]
Fixed in 77dbbb025499ee602303747d2d9e6afbe3015570.  Sorry this took so long!
